website/addons/dryad/model.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+
+from modularodm import fields
+
+from framework.exceptions import HTTPError
+
+from website.addons.base import AddonUserSettingsBase, AddonNodeSettingsBase
+from website.addons.base import StorageAddonBase
+
+from website.addons.dryad import settings as dryad_settings
+#from website.addons.dryad import serializer as serializer
+
+from pyDryad import Dryad
+
+
+class AddonDryadUserSettings(AddonUserSettingsBase):
+
+    def delete(self, **kwargs):
+        super(AddonDryadUserSettings, self).delete()
+
+    @property
+    def has_auth(self):
+        return True
+
+
+class AddonDryadNodeSettings(StorageAddonBase, AddonNodeSettingsBase):
+    """
+        A Dryad node is a collection of packages. Each package is specified by a DOI, and the title is saved automatically
+    """
+    dryad_package_doi = fields.StringField()
+    complete = True
+    has_auth = True
+    provider_name = 'dryad'
+
+    user_settings = fields.ForeignField(
+        'addondryadusersettings', backref='authorized'
+    )
+
+    def delete(self, **kwargs):
+        self.dryad_package_doi = None
+        super(AddonDryadNodeSettings, self).delete()
+
+    @property
+    def folder_name(self):
+        #get the name from dryad
+        return ''
+
+    def serialize_waterbutler_credentials(self):
+        return {'storage': {}}
```

Comment:
Yea, I wasn't sure what it put into the credentials and settings. As far as I can tell, these two are separated for convenience (or organization, whatever).  Since we're pulling in only the first version of this addon, I'm leaving them in.